code/modules/clothing/head/helmet.dm

```diff
@@ -42,15 +42,21 @@
 	name = "head of security hat"
 	desc = "The hat of the Head of Security. For showing the officers who's in charge."
 	icon_state = "hoscap"
-	armor = list(melee = 65, bullet = 30, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
-	flags_inv = HIDEEARS
 
 /obj/item/clothing/head/helmet/HoS/dermal
 	name = "dermal armour patch"
 	desc = "You're not quite sure how you manage to take it on and off, but it implants nicely in your head."
 	icon_state = "dermal"
+	armor = list(melee = 65, bullet = 30, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)

```

Comment:
Why? It's gone.

Reply:
They're just given that in case someone wants to spawn in dermal patches. Their armor value never changed.